CHANGELOG.md

```diff
@@ -3,6 +3,82 @@
 Full documentation for MIGraphX is available at
 [https://rocmdocs.amd.com/projects/AMDMIGraphX/en/latest/](https://rocmdocs.amd.com/projects/AMDMIGraphX/en/latest/).
 
+## MIGraphX 2.10 for ROCm 6.2.0
+
+### Additions
+
+* Added support for ONNX Runtime MIGraphX EP on Windows
+* Added FP8 Python API 
+* Added examples for SD 2.1 and SDXL
+* Improved Dynamic Batch to support BERT
+* Added a `--test` flag in migraphx-driver to validate the installation
+* Added support for ONNX Operator: Einsum
+* Added uint8 support in ONNX Operators
+* Introduced Split-K as a performance option
```

Comment:
Enabled Split-k kernel configurations for performance improvements. 

It is not an option. It is enabled by default